include/pcmki/pcmki_output.h

```diff
@@ -28,6 +28,8 @@ void pcmk__out_epilogue(pcmk__output_t *out, xmlNodePtr *xml, int retval);
  */
 void pcmk__register_lib_messages(pcmk__output_t *out);
 
+void pcmk__register_sched_messages(pcmk__output_t *out);
+
```

Comment:
Separating the messages into their own file is fine, but I'd rather have a single register function per library

Reply:
The problem I'm trying to solve here is that I originally intended lib/pacemaker/pcmk_output.c to be where output-related stuff to support development of libpacemaker lives.  That's what pcmk__out_prologue and pcmk__out_epilogue are doing there.  At some point, I put formatted output messages in there too because <lib>_output.c is what I'm using everywhere else.

However, as the number of formatted output messages increases, I really want the two to be separate files.  So it might make sense to move pcmk__out_prologue and pcmk__out_epilogue into a new location, if you can think of a good name.  This also means all the messages can go into pcmk_output.c where the register function is, and then I won't have to pollute the namespace by exposing them in a header.

Reply:
Okay I think I have this looking the way I want.